pvlib/iotools/pvgis.py

```diff
@@ -390,9 +391,32 @@ def read_pvgis_hourly(filename, pvgis_format=None, map_variables=True):
     raise ValueError(err_msg)
 
 
+def _coerce_and_rotate_pvgis(pvgis_data, tz, year):
```

Comment:
```suggestion
def _coerce_and_rotate_tmy(tmy_data, tz):
```
Should only be used for tmy, right?  Source probably doesn't matter.

Does pvgis put out Feb 29?  I hope not.  I think it would be ok to just use 1990.

Reply:
Yes pvgis leaves out Feb 29 always even if month is 2012, a leap year

Reply:
changed name of private function to `_coerce_and_rotate_pvgis_tmy`